refnx/reflect/test/simulation.lgt

```diff
@@ -0,0 +1,57 @@
+C1 26.659000000000002 0.0
```

Comment:
not needed if the scattering lengths come from `periodictable`.

Reply:
see point 2.